README.md

```diff
@@ -43,10 +43,28 @@ when building the images, e.g.:
 
 ## TileDB-R
 
-There is an image available for [TileDB-R](https://github.com/TileDB-Inc/TileDB-R)
-which is used only for CI. This image is build and designed to facilitate CI.
+There is two images available for [TileDB-R](https://github.com/TileDB-Inc/TileDB-R).
 
-### Instructions
+### User Image
+
+The user image is meant for end users to be able to use TileDB-R.
```

Comment:
user image -->  deployment image ?

Simpler:

This image enable user to deploy TileDB R.

(Also, we decided to drop the hyphen in the TileDB R repo last week.)